src/platform/src/common/features/create-organization/components/CreateOrganizationForm.jsx

```
@@ -48,149 +37,154 @@ const CreateOrganizationForm = ({
 
-    // Clear file errors
-    if (errors.logoFile) {
-      const newErrors = { ...errors };
-      delete newErrors.logoFile;
-      setErrors(newErrors);
-    }
-
+    setErrors((prev) => {
+      const c = { ...prev };
+      delete c.logoFile;
+      return c;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)